va/tlsalpn.go

```diff
@@ -58,36 +61,46 @@ func certAltNames(cert *x509.Certificate) []string {
 
 func (va *ValidationAuthorityImpl) tryGetChallengeCert(
 	ctx context.Context,
-	identifier identifier.ACMEIdentifier,
-	tlsConfig *tls.Config,
+	ident identifier.ACMEIdentifier,
 ) (*x509.Certificate, *tls.ConnectionState, core.ValidationRecord, error) {
-
-	allAddrs, resolvers, err := va.getAddrs(ctx, identifier.Value)
 	validationRecord := core.ValidationRecord{
-		DnsName:           identifier.Value,
-		AddressesResolved: allAddrs,
+		DnsName:           ident.Value,
+		AddressesResolved: []net.IP{},
```

Comment:
Don't fill in AddressesResolved and ResolverAddrs with their zero values. Just leave them blank.